bower.json

```diff
@@ -23,26 +23,24 @@
     "test"
   ],
   "dependencies": {
-    "purescript-aff": "^4.0.0",
-    "purescript-console": "^3.0.0",
-    "purescript-eff": "^3.1.0",
```

Comment:
Okay, I'll add `purescript-effect`. Just wondering why do we need to provide it explicitly? `ps-aff` and `ps-console` have `purescript-effect` as a dependency already.

Reply:
I think console should actually be in dev dependencies. But it's just because I don't like transitive dependencies. If I import it in the library, then it should be in the list of dependencies.

Reply:
> I think console should actually be in dev dependencies

:+1: Moved